source/core/Array.cpp

```diff
@@ -1587,9 +1579,9 @@ VIREO_FUNCTION_SIGNATURE4(ArrayThreshold, Double, TypedArrayCoreRef, Double, Int
         _Param(0) = left;
     } else if (leftValue == rightValue) {
         _Param(0) = (left + right) / 2;
-    } else if (::isinf(rightValue)) {
+    } else if (isinf(rightValue)) {
```

Comment:
@spathiwa, I think you already mentioned in another PR that we need to keep the ::, do you remember why?

Reply:
I think there is a Mac build issue without it, don't remember details.